lib/datamodels/restaurant.dart

```diff
@@ -10,25 +9,28 @@ class Restaurant {
   final Map<String, dynamic> openingHours;
   final double latitude;
   final double longitude;
+
   final NetworkImage mainPhoto;
   // Calculate distance from user with a method
   double _distanceFromUser;
 
-  Restaurant(
-      {@required this.name,
-      this.address,
-      this.rating,
-      this.openingHours,
-      this.latitude,
-      this.longitude,
-      this.mainPhoto = Placeholders.restaurantImage})
-      :
+  Restaurant({
+    @required this.name,
+    this.address,
+    this.rating,
+    this.openingHours,
+    this.latitude,
+    this.longitude,
+    this.mainPhoto,
+  })  :
         // Don't allow restaurant to display to user if name isn't known
         assert(name != null && name != ''),
         // Both geolocation and address can't be missing at the same time
         assert(((latitude != null) && (longitude != null)) ||
             (address != null && address != ''));
 
+  //? datamodels class should only have toMap / fromMap functions, all other functions like distance from user
+  //? should be inside viewmodels instead ?
```

Comment:
we should delete this too, we can ask questions on slack

Reply:
yeah we shouldn't leave questions as comments as a way to communicate haha